src/navigation.ts

```diff
@@ -28,8 +28,8 @@ export const defaultStackNavigationOptionsV2: StackNavigationOptions = {
     fontSize: 16,
     fontFamily: 'Rubik-Medium',
   },
-  headerRightContainerStyle: {
-    paddingRight: 12,
+  headerTitleContainerStyle: {
+    marginHorizontal: 45,
```

Comment:
We avoid using margin, margin is evil.

Reply:
That header is absolute positioned by default, thats the reason why was overlapped. I'm thinking of adding a width of 70% to the full width of the screen. what do you think?

Reply:
percentages are hacks
margins are terrible hacks
react navigation is hacky
hack if you must